config/config/common-config.yaml

```
@@ -158,6 +158,19 @@ prometheus:
   tolerations: []
   affinity: {}
   nodeSelector: {}
+  ## Predictive alert if the resource usage will hit the set percentage in 3 days
+  predictLinear:
+    enabled: true
+    disklimit: 66
+    usagelimit: 95
+    ## for each cpu and memory add the node pattern for which you want to create an alert and the limit value
+    requestlimit:
+      cpu:
+        worker: 80
+        elastisys: 80
+      memory:
+        worker: 80
+        elastisys: 80
```

Could there be some issues if someone wanted a more advanced pattern here? Since the pattern is also a yaml key I think it might have some restrictions.

Would this make more sense? (with appropriate change in the alert) To make it more free for the user to write whaterver they want.
```suggestion
    requestlimit:
      cpu:
        - pattern: ".*-worker-.*"
          limit: 80
      memory:
        - pattern: ".*-worker-.*"
          limit: 80
```

---

That looks good